projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/deploy/SupportedPythonVersions.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2023-2023 VMware, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.vmware.taurus.service.deploy;
+
+import java.util.HashMap;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.springframework.beans.factory.annotation.Value;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+import lombok.extern.slf4j.Slf4j;
+
+/** Handles operations related to supported python versions for data job deployments. */
+@Component
+@RequiredArgsConstructor
+@Slf4j
+public class SupportedPythonVersions {
+
+  @Value("#{${datajobs.deployment.supportedPythonVersions}}")
+  private HashMap<String, HashMap<String, String>> supportedPythonVersions;
+
+  @Value("${datajobs.vdk.image}")
+  private String vdkImage;
+
+  @Value("${datajobs.deployment.dataJobBaseImage:python:3.9-slim}")
+  private String deploymentDataJobBaseImage;
+
```

Comment:
Why do we need these ?

Reply:
These are, where we currently set the vdk and job base images to be used, so the idea is for them to act as defaults, in case the supportedPythonVersions configuration is not set.

Reply:
Why not provide default supported python versions using quickstart-vdk as the default vdk image are the standard python containers: python:3.10 

quickstart-vdk is the default image used anyhow , we simply would change the . 

I understand if you want to keep it for backwards compatibility reasons. But we should deprecate it. We should strive to remove unnecessary elements by keeping only essential features .

I might be barking the wrong tree here.

Reply:
Removed the properties, thanks.